lib/gooddata/bricks/middleware/logger_middleware.rb

```diff
@@ -36,7 +36,7 @@ def call(params)
           log_directory = params['GDC_LOG_DIRECTORY']
           execution_id = params['GDC_EXECUTION_ID']
           FileUtils.mkpath log_directory
-          logger = Logger.new("#{log_directory}/#{execution_id}.log")
+          logger = params['GDC_EXECUTION_LOG_PATH'].nil? ? Logger.new("#{log_directory}/#{execution_id}.log") : params['GDC_EXECUTION_LOG_PATH']
```

Comment:
why assign directly params['GDC_EXECUTION_LOG_PATH'] to a logger? Is is a String or a logger?